tests/amr/data/field/refine/test_refine_field.py

```
@@ -102,6 +102,9 @@ def refine_electric(field, **kwargs):
             fine_data[::2, 1::2] = 0.5 * (coarse_data[:, 1:] + coarse_data[:, :-1])
             fine_data[1::2, 1::2] = 0.5 * (coarse_data[:, 1:] + coarse_data[:, :-1])
 
+    elif field.box.ndim == 3:
+        assert False # fix
```

Ruff

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)